.github/workflows/ci.yml

```diff
@@ -21,102 +21,191 @@ jobs:
+          #  os: ubuntu-22.04-arm
+          #  compiler: gcc-9
+          #  clang-runtime: '16'
+          #  cling: Off
+          #  cppyy: Off
+          #  coverage: true
+          #
+          #- name: ubu22-arm-gcc9-clang-repl-16-cppyy
+          #  os: ubuntu-22.04-arm
+          #  compiler: gcc-9
+          #  clang-runtime: '16'
+          #  cling: Off
+          #  cppyy: On
+          #  coverage: true
+          #
+          #- name: ubu22-arm-gcc9-clang13-cling
+          #  os: ubuntu-22.04-arm
+          #  compiler: gcc-9
+          #  clang-runtime: '13'
+          #  cling: On
+          #  cling-version: '1.0'
+          #  cppyy: Off
+          #
+          #- name: ubu22-arm-gcc9-clang13-cling-cppyy
+          #  os: ubuntu-22.04-arm
+          #  compiler: gcc-9
+          #  clang-runtime: '13'
+          #  cling: On
+          #  cling-version: '1.0'
+          #  cppyy: On
+          #  
+          - name: win2022-msvc-clang-repl-17
+            os: windows-2022
+            compiler: msvc
+            clang-runtime: '17'
+            cling: Off
+            cppyy: Off
+          #Commented out until rest of ci for Windows tested
+          #- name: win2022-msvc-clang-repl-17-cppyy
+          #  os: windows-2022
+          #  compiler: msvc
+          #  clang-runtime: '17'
+          #  cling: Off
+          #  cppyy: On
+          #
+          - name: win2022-msvc-clang-repl-16
+            os: windows-2022
+            compiler: msvc
+            clang-runtime: '16'
+            cling: Off
+            cppyy: Off
+          #Commented out until rest of ci for Windows tested
+          #- name: win2022-msvc-clang-repl-16-cppyy
+          #  os: windows-2022
+          #  compiler: msvc
+          #  clang-runtime: '16'
+          #  cling: Off
+          #  cppyy: On
+          
+          - name: win2022-msvc-cling
+            os: windows-2022
+            compiler: msvc
+            clang-runtime: '13'
+            cling: On
+            cling-version: '1.0'
+            cppyy: Off
+          #Commented out until rest of ci for Windows tested
+          #- name: win2022-msvc-cling-cppyy
+          #  os: windows-2022
+          #  compiler: msvc
+          #  clang-runtime: '13'
+          #  cling: On
+          #  cling-version: '1.0'
+          #  cppyy: On
+          #  
           - name: osx13-x86-clang-clang-repl-17
             os: macos-13
             compiler: clang
             clang-runtime: '17'
             cling: Off
             cppyy: Off
-
+          
           - name: osx13-x86-clang-clang-repl-17-cppyy
             os: macos-13
             compiler: clang
             clang-runtime: '17'
             cling: Off
             cppyy: On
-
+          
           - name: osx13-x86-clang-clang-repl-16
             os: macos-13
             compiler: clang
             clang-runtime: '16'
             cling: Off
-            cppyy: Off
-
+            cppyy: Off 
+          
```

Comment:
Let's clean up the whitespace changes if they are irrelevant.

Reply:
White space removed with latest commit.